src/client/Home.css

```diff
@@ -35,3 +35,22 @@
     transform: rotate(360deg);
   }
 }
+
+.Table {
```

Comment:
HTML is `case-insensitive`. Therefore, we prefer to naming our css with [`Lisp-case`](https://en.wikipedia.org/wiki/Letter_case#Special_case_styles)

Also, the link below is the naming convention for css.
ref: https://seesparkbox.com/foundry/bem_by_example